src/granite/querying.cr

```diff
@@ -73,28 +73,26 @@ module Granite::Querying
     first(clause, params) || raise NotFound.new("Couldn't find " + {{@type.name.stringify}} + " with first(#{clause})")
   end
 
-  # find returns the row with the primary key specified.
-  # it checks by primary by default, but one can pass
-  # another field for comparison
+  # find returns the row with the primary key specified. Otherwise nil.
   def find(value)
-    return find_by(@@primary_name, value)
+    first("WHERE #{@@primary_name} = ?", value)
```

Comment:
I think this is fine.  I can't see any downsides calling `first` instead of `find_by`.  In the end, the same query will be performed.